src/Razor/src/Microsoft.VisualStudio.RazorExtension/csharp-language-configuration.json

```diff
@@ -0,0 +1,40 @@
+﻿{
```

Comment:
It would be ideal if these were copied from a central location as part of the build process rather than duplicated, but I don't think that needs to happen here, and they don't change often so maybe not even worth it.

Reply:
I get the sentiment. Part of me wonders if we'll end up changing these a good amount though to meet VS' needs.